### PR TITLE
Rearrange tests

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx [hunk 1]
```diff
@@ -13,34 +14,39 @@ const props = {
...
   expect(wrapper.find('#stripe-form').exists()).toEqual(true)
 })
 
-it('should disable submit button by default', () => {
-  const wrapper = mount(<StripeElementsForm { ...props } />)
+it('should enable submit button when form is complete and valid', () => {
+  const wrapper = mount(<StripeElementsForm {...defaultProps} />)
   expect(wrapper.find('#stripe-submit').prop('disabled')).toEqual(true)
+
+  // TODO: this should be possible without mocking the whole component
```
Comment (damianpm) [inline spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx:30]: I think is useful to kepp the reference to the issue, at least we know where to look for updates

```suggestion
  // TODO: this should be possible without mocking the whole component
  // See discussion in https://github.com/stripe/react-stripe-js/issues/59
  // we should update this tests when the issue is updated
```

Comment (josemigallas) [reply spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx:30]: OK but that issue is not directly related to this particular test. In fact this one should be doable right now 🤔

Comment (josemigallas) [reply spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx:36]: How about this?
https://github.com/3scale/porta/pull/2340/files#diff-acd794a07a447a8d9c0e82861a6759477c0eb5cdcd27e2fbe596fdbeec23ac21R20-R25

Comment (damianpm) [reply spec/javascripts/PaymentGateways/StripeFormWrapper.spec.jsx:36]: good for now

**Review by damianpm — APPROVED** [review]

**Pull request merged**